src/bootstrap/src/utils/exec.rs

```diff
@@ -209,15 +211,22 @@ impl<'a> BootstrapCommand {
         exec_ctx.as_ref().start(self, OutputMode::Capture, OutputMode::Print)
     }
 
-    /// Provides access to the stdlib Command inside.
-    /// FIXME: This function should be eventually removed from bootstrap.
-    pub fn as_command_mut(&mut self) -> &mut Command {
-        // We proactively mark this command as executed since we can't be certain how the returned
-        // command will be handled. Caching must also be avoided here, as the inner command could be
-        // modified externally without us being aware.
-        self.mark_as_executed();
-        self.do_not_cache();
-        &mut self.command
+    /// Spawn the command in background, while capturing and returns a handle to stream the output.
+    #[track_caller]
+    pub fn stream_capture(
+        &'a mut self,
+        exec_ctx: impl AsRef<ExecutionContext>,
+    ) -> Option<StreamingCommand> {
+        exec_ctx.as_ref().stream(self, OutputMode::Capture, OutputMode::Capture)
+    }
+
+    /// Spawn the command in background, while capturing and returning stdout, and printing stderr.
+    #[track_caller]
+    pub fn stream_capture_stdout(
```

Comment:
We only need `stream_capture_stdout` right now, right? So we can remove `stream_capture`. Reading both stdout and stderr in a streaming fashion while avoiding deadlocks is relatively complicated anyway, and we don't need it currently in bootstrap.

Reply:
Looks like the `stream_capture` method is still there?